components/CustomError.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Paper, Alert, Button } from '@mui/material'
+
+const CustomError = ({ title, message }) => {
+  return (
+    <div style={{ textAlign: 'center' }}>
+      <Paper elevation={ 4 } sx={{ p: 1, backgroundColor: 'white' }}>
+        <h1>{ title }</h1>
+      </Paper>
+      < br />
+      <Paper elevation={ 4 }>
+        <Alert variant="filled" severity="error">{ message }</Alert>
```

Comment:
We use `Alert` when we want to send a direct message to the user, but not when they get the wrong page. So let's keep it simple and use a `Card` here too.